src/Broadway/CommandHandling/ClosureCommandHandler.php

```diff
@@ -31,13 +32,22 @@ public function add(\Closure $handler): void
         $reflection = new \ReflectionFunction($handler);
         $reflectionParams = $reflection->getParameters();
 
-        if (!isset($reflectionParams[0]) || !$reflectionParams[0]->getClass()) {
```

Comment:
is this needed because of PHP8?

Reply:
getClass is not supported by PHP8 so we need to change some parts for deprecation problems .

Reply:
Ok, can you add a test for this?

Reply:
I thought that this class was already covered by `ClosureCommandHandlerTest`, which behavior do you think that needs to have a new test?

Reply:
It is tested indeed. I do find this piece of code hard to undertand.